CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 ## CHANGELOG
 
-### 3.0.0 (2021-03-18)
+### 3.1.0 (in progress)
+**SDK**
+- **Feature:** Added name and email as optional to Contact.
```

Comment:
I am not sure, if we can define this change in SDK as a feature. I think, it can be written, 

**Change:** Added `name` and `email` as optional fields into `Contact` data class.